src/st_stuff.c

```diff
@@ -318,10 +322,687 @@ int             st_keyorskull[3];
 // a random number per tick
 static int      st_randomnumber;
 
+static sbardefs_t *sbardef;
+
+static patch_t **facepatches = NULL;
+
 //
 // STATUS BAR CODE
 //
 
+static void LoadFacePathes(void)
```

Comment:
Should be `LoadFacePatches`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1916/commits/c40caecd5428fa13fd15c884b0a4be3e2b083e76